packages/botonic-react/src/webchat/message-list/scroll-button.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext, useEffect, useState } from 'react'
+
+import ArrawScrollDown from '../../assets/arrow-scroll-down.svg'
+import { WEBCHAT } from '../../constants'
+import { WebchatContext } from '../../contexts'
+import { resolveImage } from '../../util/environment'
+import { ContainerScrollButton } from './styles'
+
+interface ScrollButtonProps {
+  handleClick: () => void
+}
+
+export const ScrollButton = ({
+  handleClick,
+}: ScrollButtonProps): JSX.Element => {
+  const { getThemeProperty } = useContext(WebchatContext)
+
+  const [show, setShow] = useState(false)
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setShow(true)
+    }, 500)
+
+    return () => {
+      clearTimeout(timeoutId)
+    }
+  }, [])
```

Comment:
Doubt: Why are you delaying the display of this button?

Reply:
Because sometimes the button would appear for a moment. Maybe I didn't have all the development finished, I'll try to remove the delay and test again

Reply:
Removed works fine without this delay